site/_docs/configuration.md

```diff
@@ -32,7 +32,8 @@ class="flag">flags</code> (specified on the command-line) that control them.
     <tr class="setting">
       <td>
         <p class="name"><strong>Site Source</strong></p>
-        <p class="description">Change the directory where Jekyll will read files</p>
+        <p class="description">Change the directory where Jekyll will
+        read files</p>
```

Comment:
I don't see how this should improve the readability?
Personally I would like to see it much more like this:
``` html
<p class="description">Change the directory where Jekyll will read files</p>
```
or if the line *has* to be shorter, like this:
``` html
<p class="description">
    Change the directory where Jekyll will read files
</p>
```

Same comment for a few other occurrences.